ctera/src/main/java/ch/cyberduck/core/ctera/CteraMoveFeature.java

```diff
@@ -34,5 +36,8 @@ public void preflight(final Path source, final Path target) throws BackgroundExc
         if(!CteraTouchFeature.validate(target.getName())) {
             throw new InvalidFilenameException(MessageFormat.format(LocaleFactory.localizedString("Cannot rename {0}", "Error"), source.getName())).withFile(source);
         }
+        assumeRole(source, DELETEPERMISSION);
+        assumeRole(target, WRITEPERMISSION);
+        assumeRole(target.getParent(), target.getName(), source.isDirectory() ? CREATEDIRECTORIESPERMISSION : CREATEFILEPERMISSION);
```

Comment:
According javadoc in `CteraAttributesFinderFeature` both roles apply to directories only.

Reply:
`target.getParent()` makes it a directory?